Esteban-Salas-Posada/tarea-1/style.css

```
@@ -0,0 +1,198 @@
+html{
+    font-size: 62.5%;
+    background-color: #b1b1b1;
+    margin: 0;
+    padding: 0;
+}
+img{
+    max-width: 100%;
+}
+h1{
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+}
+p{
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+}
+.main{
+    background-color: #BBDDF6;
+    background-image: url(../img/modelo1.png);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    max-width: 400pxpx;
+    width: 95%;
+    height: 42vh;
+    margin: auto auto;
+}
+
+.header{
+    max-width: 95%;
+    max-height: 25%;
+    margin: 0 auto;
+}
+.header__title{
+    font-size: 3rem;
+    color: #7FB0D4;
+    text-shadow: 3px 1px #333;
+    text-align: center;
+}
+.header__subtitle{
+    font-size: 1rem;
+    text-align: right;
+    text-shadow: 1px 1px #fff;
+}
+
+/* ----------- PARRAFOS  ---------- */
+.magazine__parrafo--primero{
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: right;
+    margin: 50px 20px 0 0;
+}
+.magazine__parrafo--sgundo{
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: left;
+    margin: 50px 0 0 20px;
+}
+.magazine__parrafo--tercero{
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: right;
+    margin: 50px 20px 0 0;
+    font-weight: bold;
+}
+.magazine__parrafo--cuarto{
+    font-size: 1rem;
+    text-shadow: 1px 1px #b1b1b1;
+    text-align: left;
+    margin: 0px 0 0 20px;
+}
+
+/* --------- Footer -------------- */
+
+.footer{
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 30px;
+}
+.footer__img{
+    max-width: 30%;
+    margin: 0 20px;
+}
+.footer__p{
+    width: 100%;
+    margin: 0px 20px 0 0;
+    font-size: 0.5rem;
+    text-align: center;
+    color: white;
+}
+
+/* 480px --------------------------- */
+@media (min-width: 468px){
+    .main{
+        background-color: #BBDDF6;
+        background-image: url(../img/modelo1.png);
```

@esteban-salas , this path is not working for me please fix
```suggestion
    background-image: url(img/modelo1.png);
```